### PR TITLE
Add Trino Summit 2024 recap post

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Review by martint — APPROVED** [review]

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "Trino Summit 2024 resources"
+author: "Manfred Moser, Monica Miller, Anna Schibli"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2024/recap-blog-banner.png
+---
+
+What a view we had at the summit! Over 700 live attendees enjoyed the sessions
+and learned more about Trino and related use cases and projects. Now it is time
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:10]:
```suggestion
and learned more about Trino related use cases and projects. Now it is time
```

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+**Running Trino as exabyte-scale data warehouse**
+<br>&nbsp;&nbsp;&nbsp;Presented by Alagappan Maruthappan from [Netflix]({{site.baseurl}}/users.html#netflix)
+<br>&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://youtu.be/WuUS73QPuZE){:target="_blank"}
+
+---
+
+**Data lake at Wise powered by Trino and Iceberg**
+<br>&nbsp;&nbsp;&nbsp;Presented by Peter Kosztolanyi and Abdallah Alkhawatrah from [Wise](https://wise.com)
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:33]:
```suggestion
<br>&nbsp;&nbsp;&nbsp;Presented by Peter Kosztolanyi and Abdullah Alkhawatrah from [Wise](https://wise.com)
```

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+**Using Trino as a strangler fig**
+<br>&nbsp;&nbsp;&nbsp;Presented by Trevor Kennedy from [Fanduel](https://www.fanduel.com/)
+<br>&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://youtu.be/cVA5IPWdHRs){:target="_blank"}
+| [Slides]({{site.url}}/assets/blog/trino-summit-2024/trino-summit-2024-fanduel.pdf){:target="_blank"}
+
+---
+
+**A Lakehouse that simply works**
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:45]:
```suggestion
**A lakehouse that simply works**
```

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+**Lessons and news from the AI world for Trino**
+
+* Manfred Moser, panel moderator and Trino maintainer at [Starburst]({{site.baseurl}}/users.html#starburst)
+* Gunther Hagleitner, CEO and Co-founder at [Waii](https://waii.ai/)
+* Rong Rong, Software Engineer at [CharacterAI](https://character.ai/)
+* William Chang, Co-founder and CTO of [Canner]({{site.baseurl}}/users.html#canner) and
+  [WrenAI]({{site.baseurl}}/ecosystem/client-application.html#wren-ai)
+* Mustafa Sakalsiz, Founder and CEO at [Peaka]({{site.baseurl}}/users.html#peaka)
+* Dain Sundstrom, Trino co-creator and CTO at [Starburst]({{site.baseurl}}/users.html#starburst)
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:82]:
```suggestion
* Dain Sundstrom, Trino co-creator
```

Comment (monimiller) [reply _posts/2024-12-18-trino-summit-2024-quick-recap.md:82]: your choice here - whatever is more respectful to Dain

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://youtu.be/gobl6PhIWeE){:target="_blank"}
+
+## Day 2 sessions
+
+**Trino for Observability at Intuit**
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:88]:
```suggestion
**Trino for observability at Intuit** 
```

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+**Wvlet: A new flow-style query language for functional data modeling and interactive analysis**
+<br>&nbsp;&nbsp;&nbsp;Presented by Taro L. Saito from [Treasure Data]({{site.baseurl}}/users.html#treasuredata)
+<br>&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://youtu.be/ot7z7J6h9rM){:target="_blank"}
+| [Slides]({{site.url}}/assets/blog/trino-summit-2024/trino-summit-2024-wvlet.pdf){:target="_blank"}
+
+---
+**Securing data pipelines at the storage layer**
+<br>&nbsp;&nbsp;&nbsp;Presented by Andrew MacKay from [Superna](https://superna.io/).
+<br>&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:139]:
```suggestion
<br>&nbsp;&nbsp;&nbsp;<i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://youtu.be/Lxr4Rzn27cw){:target="_blank"}
```

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+| [Slides]({{site.url}}/assets/blog/trino-summit-2024/trino-summit-2024-superna.pdf){:target="_blank"}
+
+---
+**Empowering Pharmaceutical Drug Launches with Trino-Powered Sales Data Analytics**
```
Comment (monimiller) [inline _posts/2024-12-18-trino-summit-2024-quick-recap.md:143]:
```suggestion
**Empowering pharmaceutical drug launches with Trino-powered sales data analytics**
```

**Comment by monimiller** [comment]
minor nits then LGTM

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md (hunk 1 above)
Comment (mosabua) [reply _posts/2024-12-18-trino-summit-2024-quick-recap.md:10]: if I do that.. it needs a dash ;-)

File: _posts/2024-12-18-trino-summit-2024-quick-recap.md (hunk 6 above)
Comment (mosabua) [reply _posts/2024-12-18-trino-summit-2024-quick-recap.md:139]: haha.. forgot to paste that one

**Comment by mosabua** [comment]
All tested and ready to go. I will do follow up PRs for the user pages entries and so on later

**Pull request merged**